frontend/src/app/api/BaseApi.ts

```diff
@@ -15,32 +15,26 @@ import {
   ValidationError,
 } from "src/errors";
 import { QueryParamData } from "src/services/search/searchfetcher/SearchFetcher";
-// TODO (#1682): replace search specific references (since this is a generic API file that any
```

Comment:
I believe I've accomplished this now, though the ticket mentioned here was already closed.